practice-02/README.md

```diff
@@ -0,0 +1,58 @@
+# JavaScript #Practice-02
+## Overview
+#### INTRODUCE

```

Comment:
Please re write the introduce that help reader can catch up what are you doing here quickly?

Reply:
it's not a feature. It's all about practice